akka-bbb-apps/src/main/scala/org/bigbluebutton/core/db/CaptionLangDAO.scala

```diff
@@ -22,14 +22,14 @@ class CaptionLocaleTableDef(tag: Tag) extends Table[CaptionLocaleDbModel](tag, N
 }
 
 object CaptionLocaleDAO {
-  def insertOrUpdateCaptionLocale(meetingId: String, locale: String, captionType: String, ownerUserId: String) = {
+  def insertOrUpdateCaptionLocale(meetingId: String, locale: String, captionType: String, userId: String) = {
     DatabaseConnection.db.run(
       TableQuery[CaptionLocaleTableDef].insertOrUpdate(
         CaptionLocaleDbModel(
           meetingId = meetingId,
           locale = locale,
           captionType = captionType,
-          ownerUserId = ownerUserId,
+          ownerUserId = userId,

```

Comment:
Maybe this column should be renamed to `createdBy` instead of `ownerUserId`, what do you think?